src/Magnum/Platform/WindowlessEglApplication.h

```diff
@@ -134,6 +134,7 @@ class WindowlessEglContext {
     private:
         #ifndef MAGNUM_TARGET_WEBGL
         bool _sharedContext = false;
+        static constexpr UnsignedInt MAX_DEVICES = 128;
```

Comment:
This is used only in the cpp, so I would move that there into a unnamed namespace, named `MaxCudaDevices`.

Reply:
Will do, although it will be `MaxEGLDevices` as non-cuda devices will also show up in the query of all devices.